### ISSUE TITLE
[bug] Creeper++ nullifier cutoff issues

### ISSUE THREAD

**Issue by Regallion** [KnuckleCracker/CW4-bug-tracker#191]
The nullifier cutoff for Creeper++  has a few issues that i believe prevent it from working as intended.

Issue 1

The nullifier cutoff is not mentioned to the player.
![image](https://user-images.githubusercontent.com/40322714/99190439-976c9800-2777-11eb-938b-86a5605e83f0.png)
In this screenshot, you can see the nullifiers turn of at 10. However, there is no warning to the player that this would happen. Since the cutoff is adjustable, that means that nullifiers cutting off is near-guaranteed to be a surprise. This is, in my opinion, severely detrimental to player's ability to plan.
The fact that a player can save and then press shift+e to check also means that there is no point in keeping that information hidden, as it just adds unneeded busywork when working on tactics.

Issue 2

The nullifier cutoff is invisible in practice. Yes, the nullifiers stop working.
However, not only can you still build the nullifiers, they will even show their ranges as green around the units. This, combined with issue 1, means that a mean mapmaker can set the nullifier cutoff to 1 and the player will not know that nullifiers do not work until they build 1, which can also lead to restarts due to strategy failure. This is not ideal.

Issue 3

This is more of a bug

1.  Set nullifier cutoff on a map
2. Make it so that Creeper++ passes that cutoff
3. Change creeper++ timer in editor, so that the multiplier is back below cutoff
4. The cutoff remains in effect

**Comment by knucracker** [comment]
I've updated the c++ display to show the nullifier fail level. I've also made it so that nullifiers detonate and explode after the level it hit. Previously they entered a 'dead' state that was not reversible. "Blown up" is probably the more explicit dead state that people are familiar with :)  

The display for the cutoff level will turn red after it is exceeded.  I have not prevents building new nullifiers. That's more trouble and not a high enough priority. The display solves most of the issues, I think.

**Comment by jafdy** [comment]
This is perhaps not clear enough, I assumed that "10 nullifiers fail" meant that I would have to deal with nullifiers failing 10 times. Reading this issue is the first time I have heard of how this mechanic is meant to work.
Much of the game (for a slow moving player) involves working up to nullifying things so having a limit on how slow a player can be isn't obvious. Hitting this point unexpectedly is que to give up given a map can take half an hour or more to play and moving at the required speed will probably require a large strategy change.

TLDR: Nullifier fail message not sufficiently clear.
